docs/389ds/FAQ/bdb-deprecation.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "BDB backend deprecation"
+---
+
+# BDB Deprecation and using lmdb
+
+Since 389-ds-base 2.1.0, The 389 ldap directory server supports two kind of underlying database:
+
+- Berkeley Database (bdb)
+
+- Lightning Memory-Mapped Database Manager (lmdb)
+
+Starting from version 3.0.0 New instances are no more created with bdb by default but with libdb.
+
+Newly created instances are still created with bdb by default while libdb is flagged as [deprecated since Fedora 33]([Changes/Libdb deprecated - Fedora Project Wiki](https://fedoraproject.org/wiki/Changes/Libdb_deprecated)), this change is about to create lmdb instances by default.
```

Comment:
A bit of a nitpick, but I think http://www.lmdb.tech/doc/ is usually written in upper-case - `LMDB`. The same for `BDB` - https://en.wikipedia.org/wiki/Berkeley_DB .
Or even `BerkeleyDB` - as used in https://fedoraproject.org/wiki/Changes/Libdb_deprecated